docs/Changelog.md

```diff
@@ -20,6 +20,7 @@ Nussknacker versions
 * [#944](https://github.com/TouK/nussknacker/pull/903) Pluggable SchemaRegistry Cache
 * [#704](https://github.com/TouK/nussknacker/pull/704) Preloaded creator panel node icons
 * [#943](https://github.com/TouK/nussknacker/pull/943) Literal min / max validators
+* [#981](https://github.com/TouK/nussknacker/pull/981) Refactor Kafka and KafkaAvro serialization package
```

Comment:
Can it be squashed with `KafkaAvroSchemaProvider`? It looks like it was logically the same work unit, but done in two steps.

Reply:
I created only one point with information about PRs.